src/hera.cpp

```diff
@@ -44,6 +44,14 @@ using namespace HeraVM;
 
 namespace {
 
+vector<uint8_t> sentinel(vector<uint8_t> const& input)
```

Comment:
For this case you can pass by value, but that will probably change with the real metering.